### ISSUE TITLE
Clone msmbo repo into your local computer

### ISSUE THREAD

**Issue by a4ayad** [msimbo/codeofhonors-jobs-tracker-v1v2#11]
Clone msmbo repo https://github.com/msimbo/codeofhonors-jobs-tracker-v1v2 into your local computer from cmder terminal using the following command:

cd msimbo-projects
git clone git@github.com:msimbo/codeofhonors-jobs-tracker-v1v2.git
cd codeofhonors-jobs-tracker-v1v2

**Comment by a4ayad** [comment]
I cloned the repo into my local computer, created and pushed index.html

**Comment by Lamwatk13** [comment]
Clone msmbo codeofhonors-jobs-tracker-v1v2 to my local system

**Comment by jtillman83** [comment]
cloned the repo

**Comment by Lamwatk13** [comment]
cloned the repo

**Comment by Coding-Edwar** [comment]
I have cloned the repo

**Comment by a4ayad** [comment]
We made sure that everyone can push changes using VSCode into the branch feat/header of the repo on github.

We created the github branch feat/header from VSCode.

Here are the commands we used from cmder terminal:

% git branch
% git log
% git remote -v
% git checkout feat/header
% git pull 

After editing a file:

% push to the branch repo

We checked that the files are updated on the branch feat/header of the repo on github.
